Classes/common/touchdb/TDRemoteRequest.h

```diff
@@ -17,30 +18,27 @@ typedef void (^TDRemoteRequestCompletionBlock)(id result, NSError* error);
 
 /** Asynchronous HTTP request; a fairly simple wrapper around NSURLConnection that calls a
  * completion block when ready. */
-@interface TDRemoteRequest : NSObject <NSURLConnectionDelegate
-#if TARGET_OS_IPHONE || defined(__MAC_10_8)
-                                       ,
-                                       NSURLConnectionDataDelegate
-#endif
-                                       > {
+@interface TDRemoteRequest : NSObject<NSURLSessionDelegate> {
```

Comment:
I'm a bit confused here, does this class need to conform to the `NSURLSessionDelegate` protocol any more, does anything set this as a delegate for `NSURLSession`? In TDReplicator.m you have `self.session = [[CDTURLSession alloc] initWithDelegate:nil callbackThread:_replicatorThread` so my guess is that these delegate methods are never called except 'manually' from inside the `TDRemoteRequest`'s `completionHandler`